PartyPlaylists/PartyPlaylists/Models/DataModels/RoomSong.cs

```diff
@@ -2,18 +2,20 @@
 using PartyPlaylists.Models.DataModels;
 using System;
 using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
 using System.Linq;
 using System.Threading.Tasks;
 
 namespace PartyPlaylists.Models.DataModels
 {
     public class RoomSong
     {
+        [Key]
+        [Required]
         public int Id { get; set; }
-
+        [Required]
         public int RoomId { get; set; }
         public Room Room { get; set; }
-
         public int SongId { get; set; }
```

Comment:
A RoomSong object should always have an associated room, and song. For this reason, I believe the [Required] attribute should be added to the SongId property as well.

```suggestion
        [Required]
        public int SongId { get; set; }
```